app/src/main/java/ca/pkay/rcloneexplorer/Services/SyncService.java

```diff
@@ -112,34 +117,45 @@ protected void onHandleIntent(@Nullable Intent intent) {
         startForeground(PERSISTENT_NOTIFICATION_ID_FOR_SYNC, builder.build());
 
         currentProcess = rclone.sync(remoteItem, remotePath, localPath, syncDirection);
+        JSONObject stats;
+        String notificationContent = "";
+        String[] notificationBigText = new String[5];
         if (currentProcess != null) {
             try {
                 BufferedReader reader = new BufferedReader(new InputStreamReader(currentProcess.getErrorStream()));
                 String line;
-                String notificationContent = "";
-                String[] notificationBigText = new String[5];
                 while ((line = reader.readLine()) != null) {
-                    if (line.startsWith("Transferred:") && !line.matches("Transferred:\\s+\\d+\\s+/\\s+\\d+,\\s+\\d+%$")) {
-                        String s = line.substring(12).trim();
-                        notificationBigText[0] = s;
-                        notificationContent = s;
-                    } else if (line.startsWith(" *")) {
-                        String s = line.substring(2).trim();
-                        notificationBigText[1] = s;
-                    } else if (line.startsWith("Errors:")) {
-                        notificationBigText[2] = line;
-                    } else if (line.startsWith("Checks:")) {
-                        notificationBigText[3] = line;
-                    } else if (line.matches("Transferred:\\s+\\d+\\s+/\\s+\\d+,\\s+\\d+%$")) {
-                        notificationBigText[4] = line;
-                    } else if (isLoggingEnable && line.startsWith("ERROR :")){
+                    JSONObject logline = new JSONObject(line);
+                    FLog.e("LOG", logline.toString());
```

Comment:
Use the TAG variable, then we know where the log is coming from :)

Reply:
That was unnessessary and for debugging purposes, i accidentially left it so i removed it